shcDecoder/build.gradle

```diff
@@ -57,6 +57,8 @@ dependencies {
     testImplementation 'org.json:json:20210307'
 
     testImplementation "junit:junit:$versions.junit"
+    testImplementation "org.mockito:mockito-core:$versions.mockito"
+    testImplementation "io.mockk:mockk:$versions.mockk"
```

Comment:
mockk is wrapper of mockito. so do we really require mockk and mockito both?

Reply:
This should be correct, however, if I remove Mockito Core dependency tests stop working somehow.